src/App.js

```diff
@@ -1,11 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
 
 function App() {
   return (
```

Comment:
- [ ] Kindly make sure the `App.js`(main component) has the imported `Calculator.js` component for the project requirements i.e. you should first import the `Calculator.js` component in the `App.js` file then after import the `App.js` (main component) in the `index.js` file(not directly import) `Calculator.js` in the `index.js` file
e.g.:
````JavaScript
//  In the App.js file:
import Calc from './components/Calculator';
...

//  In the index.js file:
import App from './App.js';


const root = ReactDOM.createRoot(document.getElementById('root'));
root.render(
  <React.StrictMode>
    <App />
  </React.StrictMode>,
);
```